esp/src/src-react/components/controls/Chip.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from "react";
+import { styled } from '@fluentui/utilities';
+import { lighten, fade } from "@material-ui/core/styles";
+
```

Comment:
I tend to not leave gaps in the imports section - when vscode auto adds a new entry it will try and put it in the correct place, except when there are gaps and then it will add to the end.